packages/react-scripts/template-typescript/src/App.test.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 
-it('renders without crashing', () => {
+it('renders without crashing', (): void => {
```

Comment:
I'm not sure I like the idea of always defining a return type - it results in very verbose code, especially in cases where TypeScript can infer.

Reply:
A good practice in development is to minimize the number of writing types. Because the results of type inference are often more accurate than the type written by beginners.